src/traversals/bfs.jl

```diff
@@ -189,12 +189,15 @@ function is_bipartite(g::SimpleGraph, s::Int)
     return visitor.is_bipartite
 end
 
-function is_bipartite(g::SimpleGraph)
-    nvg = nv(g)
-    for v in vertices(g)
-        if !is_bipartite(g, v)
-            return false
+if VERSION >= v"0.4.0-dev+6371"
+    is_bipartite(g::SimpleGraph) = all([is_bipartite(g,v) for v in vertices(g)])
```

Comment:
Ooh. yes. You're absolutely right. I think I can do this via

`is_bipartite(g::SimpleGraph) = all(x->is_bipartite(g,x), vertices(g))`

Do you agree?